.travis.yml

```diff
@@ -13,3 +13,6 @@ script:
   - jshint --show-non-errors test/*.js || echo "Errors in jshint ignored bc it doesn't support async/await"
   - GEN_TESTS_QTY=20 yarn test test/LifToken.js test/Crowdsale.js test/CrowdsaleGenTest.js
   - yarn test test/MarketMaker.js
+after_script:
+  - chmod +x scripts/coveralls.sh
```

Comment:
I think you could add this change on the actual file in this PR so this line would not be needed. Is there any downside?

Reply:
Cool! I didn't know you can do that - I thought this was something weird about yarn. Thanks!